docs/quickstart.md

```diff
@@ -2,198 +2,441 @@
 hide:
   - navigation
 ---
+## __Overview__
 
-This tutorial will introduce the main NeMoS functionalities. This is intended for users that are 
-already familiar with the GLM framework but want to learn how to interact with the NeMoS API. 
-If you have used [scikit-learn](https://scikit-learn.org/stable/) before, we are compatible with the [estimator API](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html), so the quickstart should look 
-familiar.
+NeMoS is a neural modeling software package designed to model neural spiking activity and other time-series data
+ powered by [JAX](https://jax.readthedocs.io/en/latest/). 
 
-In the following sessions, you will learn:
+At its core, NeMoS consists of two primary modules: the __`glm`__ and the __`basis`__ module.
```

Comment:
Up to you, but I think it makes sense to end this line in a colon and then make the descriptions of the `glm` and `basis` bullet points.